HACKING.md

```diff
@@ -26,6 +28,46 @@ And lastly get Webpack and the development dependencies:
     $ npm install
     $ ./node_modules/bower/bin/bower install
 
+When relying on CI to run the test suite, this is all that is
+necessary to work on the JavaScript components of Cockpit.
+
+To actually build the Cockpit binaries themselves from source
+(including to run the integration tests locally), you will need
+additional header files and other components. Check
+`tools/cockpit.spec` for the concrete Fedora build dependencies.
+The following should work in a fresh Git clone:
+
+    $ sudo yum-builddep tools/cockpit.spec
+
+In addition, for testing, the following dependencies are required:
+
+    $ sudo yum install \
+        curl \
+        expect \
+        firewalld \
+        krb5-workstation \
+        krb5-server \
+        libvirt \
+        libvirt-client \
+        libvirt-daemon \
+        libvirt-python \
+        mock \
```

Comment:
I don't think we should add mock here (I know, it was in the original list already). You can (and should) use mock for building stuff, but it's not cockpit specific, and it's unrelated to running the tests.